recipes/python-seabreeze/build.sh

```diff
@@ -0,0 +1,15 @@
+# download and compile libseabreeze
+cd extra/libseabreeze/SeaBreeze
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Platform: Mac"
+    make logger=0 install_name="${PREFIX}/lib/libseabreeze${SHLIB_EXT}" lib/libseabreeze${SHLIB_EXT}
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+    echo "Platform: Linux"
+    # we need libusb headers on circleci
+    yum install -y libusb-devel
```

Comment:
Please add this to a `yum_requirements.txt` file in the recipe directory and drop it from here.